cmd/type.go

```diff
@@ -51,11 +52,12 @@ Optionally add a _blueprint.json file to define the default field structure for
 	Run: func(cmd *cobra.Command, args []string) {
 		typeName := args[0]
 
+		// shoud we stop here on error from either?
```

Comment:
So there can be three scenarios:
* A content/data source **with** a corresponding template.
* A content/data source **without** corresponding template.
* And a template without a data source. 

Is there any situation where adding a template or data source breaks anything, say `foo.svelte` exists without `content/foo`(or vice versa) and then you `plenti new type foo`?

Reply:
Yup, so it _should_ work in all cases:
1. content **with** template = regular node endpoint(s) you can visit
2. content **without** template = content you can pull into other nodes, but has no direct node endpoint itself
3. template **without** content = rare, but currently used to handle 404 errors. You could eject the router to handle more cases.

Completing the set by either adding a template or data source shouldn't break anything, but might surprise the user with unintended consequences. For instance let's say the user inherited a site that put the main menu links into a content type called `links`. This existed as a content source without a template so there were no endpoints for the actual links (they may have pointed to nodes created by a different content type). If the user then tries to create a new content type called `links` for a different purpose, not realizing it already existed, the effect is a template will be created and it will expose all the existing links in the content source as node endpoints. In reality this would probably be discovered pretty quickly and with git the rollback would be trivial so I'm not super worried about this usecase - just wanted to put it out there.